server/realTime/scripts/Network.js

```diff
@@ -1072,6 +1140,130 @@ Protocol.prototype.bindIO = function bindIO () {
 			});
 		});
 
+		socket.on("removeprotectedregion", function (regionId, callback) {
+			if (typeof callback !== 'function') return;
+
+			if(isNaN(regionId))
+			{
+				callback("Region id is undefined.")
+				return;
+			}
+
+			protocol.players.request('removeprotectedregion', {
+				uKey: socket.uKey,
+				room: socket.room,
+				regionId: regionId
+			}, function (err, data) {
+				callback(err, data);
+				protocol.updateProtectedRegions(socket.room);
+			});
+		});
+
+		socket.on("getmyprotectedregions", function (callback) {
+			if(!socket.userid)
```

Comment:
Why do you do this trough the realTime server? Also why is there a variable asdf and why does it get stringified?